tools/topology/topology2/production/tplg-targets-hda-generic.cmake

```diff
@@ -4,6 +4,10 @@
 list(APPEND TPLGS
 # HDMI only topology with passthrough pipelines
 "sof-hda-generic\;sof-hda-generic-idisp\;"
+# HDMI only topology with passthrough pipelines and
+# 2 or 4 DMIC, no NHLT blob included in topology
+"sof-hda-generic\;sof-hda-generic-idisp-2ch\;NUM_DMICS=2"
+"sof-hda-generic\;sof-hda-generic-idisp-4ch\;NUM_DMICS=4"
```

Comment:
Is this something we want to have in production? I.e. shipped to Linux distros by default?

Reply:
I think it's a good fallback to have if e.g. the I2S or SoundWire machine search fails. Without these files the probe would fail with a 'no topology found' error.

But indeed it's not really a mainstream production setup.